policyengine_us/variables/gov/states/wv/tax/income/subtractions/wv_social_security_benefits_subtraction.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class wv_social_security_benefits_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia social security deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        # West Virginia Personal Income Tax Forms And Instructions 2020 LINE 33
+        "https://tax.wv.gov/Documents/TaxForms/2020/it140.booklet.pdf#page=24",
+        # West Virginia Personal Income Tax Forms And Instructions 2020 LINE 32
+        "https://tax.wv.gov/Documents/TaxForms/2021/it140.booklet.pdf#page=24",
+        # West Virginia Personal Income Tax Forms And Instructions 2022 LINE 32
+        "https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf#page=25",
+        # Code of West Virginia §11-21-12 (c)(8)(A) - (c)(8)(C)
+        "https://code.wvlegislature.gov/11-21-12/",
+    )
+    defined_for = "wv_social_security_benefits_subtraction_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.wv.tax.income.subtractions.social_security_benefits
+        person = tax_unit.members
+        amount = person("social_security", period) * p.rate
+        return tax_unit.sum(amount)
```

Comment:
```suggestion
        return  social_security * p.rate
```

Reply:
I am reading the form as all social security benefits are multiplied by 35% and not individually - shouldn't change much on the output, let me know if I am wrong

Reply:
I saw that in the form the security benefits are calculated for taxpayer and spouse separately.
<img width="771" alt="Screenshot 2023-09-11 at 6 51 08 PM" src="https://github.com/PolicyEngine/policyengine-us/assets/90161322/aaec3451-5746-474a-8b21-b5110b747f4d">



And they are added together when calculate total subtractions. So I do it this way.
<img width="752" alt="Screenshot 2023-09-11 at 6 51 34 PM" src="https://github.com/PolicyEngine/policyengine-us/assets/90161322/58f9e91d-1a74-43f5-9b07-3f1c6811f942">

Reply:
Is it only for head and spouse? If so we should specify that (exclude dependents)

Reply:
But we can still consolidate that as head and spouse ss multiplied by 35%